### DISCUSSION TITLE
Webhook problem in version v0.9.0-alpha from 1 to 3

### DISCUSSION THREAD

**Discussion by hainguyenkt98** [ory/kratos#2369] [Q&A] [answered]
Webhook not working in password method in register flow, oicd flow still works (version v0.9.0-alpha from 1 to 3)

**Comment by vinckr** [comment]
Hello @hainguyenkt98 
Sorry you're having trouble. We'll need more information in order to figure out what's happening here.
Can you provide some additional information, such as what were you trying to do when the error occurred?
Your setup, additional log files, your configuration and any other information that you think is helpful also helps us to solve your issue more quickly!

**Reply by hainguyenkt98** [reply]
I resolved it yesterday by changing the order of elements in the hooks array of password method. I move web_hook go to the head of the array, so it's working. It looks like the below the attachment
![Screenshot_20220407_171348](https://user-images.githubusercontent.com/25980787/162339125-2aed95b2-e1b3-43d8-a995-c88c0d7f4886.png).
If I swap the position of web_hook and session, it is not working again.

**Reply by vinckr** [reply]
Hm that should not be the case! 
Would you be willing to open [a bug report](https://github.com/ory/kratos/issues/new?assignees=&labels=bug&template=BUG-REPORT.yml) for this?
Will also see if I can reproduce it here.

**Reply by hainguyenkt98** [reply]
OK. Thanks for your response. I think I'm going to make a bug report for this problem.

**Reply by vinckr** [reply]
Can you also post your answer https://github.com/ory/kratos/discussions/2369#discussioncomment-2527402 as a extra comment, so we can select it as answer to this discussion? 
Otherwise it is nested as reply to me, thanks :-)

**Comment by vinckr** [comment, marked as answer]
quoting @hainguyenkt98 
> I resolved it yesterday by changing the order of elements in the hooks array of password method. I move web_hook go to the head of the array, so it's working. It looks like the below the attachment ![Screenshot_20220407_171348](https://user-images.githubusercontent.com/25980787/162339125-2aed95b2-e1b3-43d8-a995-c88c0d7f4886.png). If I swap the position of web_hook and session, it is not working again.
